packages/dx-react-scheduler-material-ui/src/templates/current-time-indicator/indicator.jsx

```diff
@@ -1,41 +1,53 @@
 import * as React from 'react';
+import { styled } from '@mui/styles';
 import * as PropTypes from 'prop-types';
-import makeStyles from '@mui/styles/makeStyles';
 import classNames from 'clsx';
 
-const useStyles = makeStyles(theme => ({
-  line: {
-    height: '2px',
-    width: '100%',
-    transform: 'translate(0, -1px)',
-  },
-  circle: {
+const PREFIX = 'Indicator';
+
+export const classes = {
+  line: `${PREFIX}-line`,
+  circle: `${PREFIX}-circle`,
+  nowIndicator: `${PREFIX}-nowIndicator`,
+};
+
+const StyledDivCircle = styled('div')(({ theme, topValue }) => ({
+  [`&.${classes.circle}`]: {
     width: theme.spacing(1.5),
     height: theme.spacing(1.5),
     borderRadius: '50%',
     transform: 'translate(-50%, -50%)',
   },
-  nowIndicator: {
+  [`&.${classes.nowIndicator}`]: {
```

Comment:
This class has the same styles in two styled component. The function with the styles can be moved outside and used

Reply:
Done